### ISSUE TITLE
Severe Judder

### ISSUE THREAD

**Issue by marsman12019** [mattebb/ofxOculusDK2#4]
There's pretty severe judder from the orientation tracking (on my build at least), seen here: ![2015-05-20 11_15_38](https://cloud.githubusercontent.com/assets/3644972/7729199/8e55cec2-fee1-11e4-817a-d9e3ca309c7b.gif)

Solved in my case by setting `bNoMirrorToWindow` to `true` and `bTimeWarp` to `false` in `ofxOculusDK2.cpp`.

**Comment by mattebb** [comment]
Strange that that fixes it... NoMirrorToWindow should only have an effect in Direct render mode. Are you using that? What OS are you using?

I had some weird judder effects here, and took a combination of some of the hmd capabilities toggled to get it under control.

First things to check/test though:
* Is it Vsynced to your DK2 display (75Hz) (some OSes require the DK2 to be the primary monitor for this). Is vsync on in OF? Or are you using the NoVsync hmd capability in the Oculus API?
* Is your frame rate actually consistent at 75FPS?

PS. All those hmdcaps/distortioncaps are now toggleable at run time using functions exposed  the OF addon. You don't need to edit the addon code itself for that any more.

**Comment by marsman12019** [comment]
(I just noticed all those functions right after I posted. That makes everything so much easier.)

I just tried it only with TimeWarp off, and that worked just the same. As far as I know, OSX (I'm on 10.10) doesn't have support for Direct Render Mode, right?

I've set `ofSetVerticalSync(false)` and flipping `oculusRift.setNoVsync()` does nothing. I'll do a bunch of tests with those modes and changing the primary display to confirm.

I'm not sure how to accurately check the frame rate on the Oculus? `ofGetFrameRate()` is giving me a consistent 45fps — I believe I'm being limited by the framerate of the cameras I'm using. The Oculus' display is set to 75hz, however.
